### PR TITLE
Alt-clicking abandoned crates no longer makes them explode.

### PR DESCRIPTION
**Pull request by harmonyn**
Fixes https://github.com/tgstation/-tg-station/issues/17421.

This feature was too fun to leave in.

### CODE REVIEW

File: code/modules/mining/abandoned_crates.dm [hunk 1]
```diff
@@ -174,6 +174,11 @@
 /obj/structure/closet/crate/secure/loot/attack_animal(mob/user)
 	boom(user)
 
+/obj/structure/closet/crate/secure/loot/AltClick(mob/living/user)
+	if(user.incapacitated() || !Adjacent(user) || !ishuman(user))
```
Comment (xxalpha) [inline code/modules/mining/abandoned_crates.dm]: Use iscarbon so monkeys can do it?

Comment (PKPenguin321) [inline code/modules/mining/abandoned_crates.dm]: I think just istype(user) would work?

Comment (bgobandit) [inline code/modules/mining/abandoned_crates.dm]: Nope -- it brings up the "enter code" screen but produces a "You don't have the dexterity" message if you enter anything.

**Comment by Shadowlight213** [comment]
needs fun removal tag

**Comment by KorPhaeron** [comment]
canusetopic

**Pull request merged**